detox/test/android/build.gradle

```diff
@@ -38,23 +43,25 @@ buildscript {
     }
 }
 
-allprojects {
-    repositories {
-        maven {
-            url "$rootDir/../../node_modules/react-native/android"
-        }
-        maven {
-            url("$rootDir/../node_modules/jsc-android/dist")
-        }
-        mavenCentral {
-            // We don't want to fetch react-native from Maven Central as there are
-            // older versions over there.
-            content {
-                excludeGroup "com.facebook.react"
+if (!rnInfo.isRN71OrHigher) {
+    allprojects {
+        repositories {
+            maven {
+                url "$rootDir/../../node_modules/react-native/android"
+            }
+            maven {
+                url("$rootDir/../node_modules/jsc-android/dist")
+            }
+            mavenCentral {
+                // We don't want to fetch react-native from Maven Central as there are
+                // older versions over there.
+                content {
+                    excludeGroup "com.facebook.react"
+                }
             }
+            google()
+            mavenLocal()
         }
-        google()
```

Comment:
Makes sense, although it might be a good idea to keep `google()` and `mavenLocal()` even in RN ≥71